api/src/test/resources/sampleFeedConfiguration2.json

```diff
@@ -1,11 +1,16 @@
-[ {
-  "openMrsClass" : "org.openmrs.TEST2",
-  "enabled" : false,
-  "title" : "Title2",
-  "category": "patient",
-  "linkTemplates" : {
-    "rest" : "openmrs/ws/rest/v1/patient{uuid}?v=full",
-    "fhir" : "openmrs/ws/fhir/v1/patient{uuid}?v=full"
-  },
-  "feedWriter" : "custom"
-} ]
\ No newline at end of file
+{
```

Comment:
Maybe rename it to defaultBeanConfiguration, and the other one to custom...

Reply:
Good idea, thanks